### ISSUE TITLE
gulp.dest is not respected when the proto file is not directly in the root directory

### ISSUE THREAD

**Issue by gegles** [cbrand/gulp-protobufjs#1]
Great plugin, btw.

Consider this task:

``` javascript
var gulp = require('gulp');
var gulpprotobuf = require('gulp-protobufjs');

gulp.task('proto-task', function () {
  return gulp.src('config/my-proto.proto')
    .pipe(gulpprotobuf())
    .pipe(gulp.dest('out/'));
});
```

The my-proto.proto.js file is not created under "out/" but simply at the root of the project.

Let me know if I am missing something or if you would be ok fixing this.

Cheers. G.

**Comment by cbrand** [comment]
Hey thanks for the feedback. Happy that someone uses it.

I just needed it for a side project and didn't seem to test it thoroughly enough. The fix is pushed and I'll take a look into #2 before putting a new version on npm.

**Comment by gegles** [comment]
Wow, talk about quick turn around! This is awesome. Thanks. G.
